example/otel-collector/Makefile

```diff
@@ -0,0 +1,34 @@
+namespace-k8s:
+	k apply -f k8s/namespace.yaml
```

Comment:
This needs to be `kubectl` instead of `k`

Reply:
Right, missed that one. Changed to kubectl.